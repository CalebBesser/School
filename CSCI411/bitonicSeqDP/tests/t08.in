24
-5
3
-2
13
7
-22
4
16
9
19
-3
21
-15
18
5
17
-6
13
7
3
15
-4
11
23
findBitonic
