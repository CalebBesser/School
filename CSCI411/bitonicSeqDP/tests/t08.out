-5 3 13 16 19 21 18 17 13 7 3 -4 
